.npmrc

```diff
@@ -1,13 +1,7 @@
-# Don't create or read package-lock.json
-package-lock=false
-# Use exact versions of dependencies vs. semver minor updates
-save-exact=true
```

Comment:
With this change, we might be able to switch to using `^x.y.z` style deps in `package.json` moving forward.